4eliminate.tex

```diff
@@ -6,7 +6,7 @@ \subsection{Eliminate earth orbiting satellites} \label{sec:4eliminate}
 will be determined by comparison with an appropriate catalog or catalogs at the \gls{USDF}.
 Any object that corresponds to an object in this catalog will not be released in the  prompt
 alerts or stored in the prompt alert database.
-Any catalog that SLAC creates noting these corresponding identifications will be held at the OFFICIAL USE ONLY level within the \gls{USDF} at \gls{SLAC} and have appropriate export-controls and \gls{FOIA} exemptions.
+SLAC will not create any catalogs noting these corresponding notifications.
```

Comment:
Is "SLAC" the cognizant party here?

Reply:
You can see from this that is the party that DOE holds responsible...